plugins/lsp/language-servers/default.nix

```
@@ -515,6 +515,12 @@ with lib; let
       serverName = "prolog_ls";
       package = pkgs.swiProlog;
     }
+    {
+      name = "purescript-language-server";
+      description = "purescript-language-server for PureScript";
+      serverName = "purescriptls";
+      package = pkgs.nodePackages.purescript-language-server;
+    }
```

```suggestion
    {
      name = "purescriptls";
      description = "purescriptls for PureScript";
      package = pkgs.nodePackages.purescript-language-server;
    }
```
We use the name from `nvim-lspconfigs` whenever possible.